docs/configuration.rst

```diff
@@ -203,4 +203,6 @@ The list of statistics parameters that can be modified is:
 +-------------------------------------------+--------------------------------------------------------------------------------------+
 | PERCENTILES_TO_CHART                      | The list of response time percentiles for response time chart                        |
 +-------------------------------------------+--------------------------------------------------------------------------------------+
+| PERCENTILES_TO_STATISTICS                 | The list of response time percentiles for response time statistics                   |
```

Comment:
Suggest you change this to "list of response time percentiles to show in the statistics table"

Reply:
instead maybe add something about this and PERCENTILES_TO_CHART being UI settings (just to make it clear)